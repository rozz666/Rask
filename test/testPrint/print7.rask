main() -> void
{
    print(5 5;
}

