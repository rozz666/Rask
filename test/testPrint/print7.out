print7.rask:3:13: error: missing closing parenthesis ')'

